events/wc_dark_cult_events.txt

```diff
@@ -1003,81 +1026,117 @@ province_event = {
+	}
 	border = GFX_event_narrative_frame_intrigue
 
 	is_triggered_only = yes
 
 	immediate = {
 		# Used in scripted effects
 		save_event_target_as = target_settler
-		
+
 		religion = old_gods_worship
 		clear_event_target = yes
-		
-		remove_trait = travelling
-		
-		c_hivezora = {
-			# event_target:target_settler and its people settle here
-			settle_as_target_settler_effect = yes
+		any_courtier = {
+			limit = {
+				culture = twilights_hammer
+				can_switch_religion_character_trigger = yes
+			}
+			religion = old_gods_worship
 		}
-		
-		# If you're not independent ruler, gives old titles to liege
-		give_old_titles_to_heir_effect = yes
-		
-		d_crystalvale = {
-			any_direct_de_jure_vassal_title = {
+
+		remove_trait = travelling
+		diplomatic_immunity = no
+
+		if = {
+			limit = { has_global_flag = twilights_hammer_kalimdor }
+			c_southmoon = {
 				# event_target:target_settler and its people settle here
 				settle_as_target_settler_effect = yes
 			}
+
+			# If you're not independent ruler, gives old titles to liege
+			give_old_titles_to_heir_effect = yes
+
+			c_southmoon = {
+				# Declares war against player
+				target_settler_declare_war_effect = yes
+			}
 		}
-		d_scarab_wall = {
-			any_direct_de_jure_vassal_title = {
+		else = {
+			c_strand_of_ancients = {
 				# event_target:target_settler and its people settle here
 				settle_as_target_settler_effect = yes
 			}
+
+			# If you're not independent ruler, gives old titles to liege
+			give_old_titles_to_heir_effect = yes
+			
+			c_strand_of_ancients = {
+				# Declares war against player
+				target_settler_declare_war_effect = yes
+			}
 		}
-		
-		c_hivezora = {
-			# Declares war against player
-			target_settler_declare_war_effect = yes
-		}
-		
 		clear_settler_flag_effect = yes
-		
-		# Spawns army for start
-		spawn_colonial_troops_effect = yes
 	}
 
-	# I'll spread its will across Kalimdor!
+	# I'll spread its will across Kalimdor/Northrend!
 	option = {
 		name = EVTOPTA_WCDAC_5040
 
-		add_artifact = hammer_of_the_twilight
-		leave_society = yes
+		if = {
+			limit = {
+				potential_shadow_priest_wo_training_trigger = yes
+				is_magic_class_trigger = yes
+			}
+			hidden_effect = {
+				add_character_modifier = { name = training_self_shadow_priest duration = -1 }
+				recount_experience_effect = yes
+				narrative_event = { id = WCCLS.1300 }
+			}
+		}
+		if = { limit = { NOT = { trait = zealous } add_trait = zealous } }
+
+		add_35_void_corruption_effect = yes
+
 		set_global_flag = twilights_hammer_cult_active_flag
 		join_society_twilights_hammer_cult_effect = yes
 		hidden_effect = {
 			set_society_grandmaster = yes # Set character as Grand Master
 			add_society_currency_massive_effect = yes # Give the Grand Master a start-up capital to induct members
 		}
 
-		add_35_void_corruption_effect = yes
-
 		hidden_effect = {
-			activate_title = { title = d_twilights_hammer status = no }
-			d_twilights_hammer = { destroy_landed_title = THIS }
```

Comment:
Holy Light Cho'gall rules the clan being kicked from the society.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/86471613-c2b59400-bd4e-11ea-840e-fe641a3a5494.png)
![image](https://user-images.githubusercontent.com/46384992/86471647-d4973700-bd4e-11ea-9fb9-e2deab65598c.png)

</details>